src/user/authReducers.js

```diff
@@ -0,0 +1,38 @@
+import { fromJS } from 'immutable';
+
+import {
+  ACCOUNT_ADD_SUCCEEDED,
+  LOGIN_SUCCEEDED,
+  LOGOUT,
+} from './userActions';
+
+const initialState = fromJS({
```

Comment:
How will we handle multiple accounts with a single auth reducer?

That was the original motivation for having auth information inside of each account. Eventually we'd like to add an account switcher that lets the user toggle between multiple logged in Zulip servers.